.env.example

```diff
@@ -6,3 +6,4 @@ APPROVED_PREMISES_ROOT_PATH=approved-premises
 HMPPS_AUTH_USERNAME=
 HMPPS_AUTH_PASSWORD=
 E2E_USER_TO_ADD=BERNARD.BEAKS
+E2E_EMERGENCY_USER_TO_ALLOCATE_TO="JIM SNOW"
```

Comment:
There's a bit of a gotcha here in the way we are specifying:

- Delius **username** for `E2E_USER_TO_ADD` and 
- Delius **name** for `E2E_EMERGENCY_USER_TO_ALLOCATE_TO`

Should we try to stress difference, e.g. with:

- `E2E_DELIUS_USERNAME_TO_ADD`
- `E2E_EMERGENCY_ASSESSOR_NAME_TO_ALLOCATE_TO`

Reply:
yep, good spot